src/transactions/typeTag/typeTag.ts

```diff
@@ -38,6 +38,9 @@ export abstract class TypeTag extends Serializable {
         return TypeTagU32.load(deserializer);
       case TypeTagVariants.U256:
         return TypeTagU256.load(deserializer);
+      case TypeTagVariants.Generic:
+        // This is specifically a placeholder and does not represent a real type
```

Comment:
a bit more explanation on why?

Reply:
So, this is not a real type (only for the ABI).  We will have to do some smarter stuff around the generics, which actually I have to go back and fix something here first.